release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -0,0 +1,434 @@
+[id="ocp-4-7-release-notes"]
+= {product-title} {product-version} release notes
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid
+cloud application platform for deploying both new and existing applications on
+secure, scalable resources with minimal configuration and management overhead.
+{product-title} supports a wide selection of programming languages and
+frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Kubernetes, {product-title}
+provides a more secure and scalable multi-tenant operating system for today's
+enterprise-class applications, while delivering integrated application runtimes
+and libraries. {product-title} enables organizations to meet security, privacy,
+compliance, and governance requirements.
+
+[id="ocp-4-7-about-this-release"]
+== About this release
+
+// TODO: Update k8s link once there is a version-specific URL for the Kubernetes release
+(link:https://access.redhat.com/errata/RHBA-2020:1234[RHBA-2020:1234]) is now
+available. This release uses link:https://kubernetes.io/docs/setup/release/notes/[Kubernetes 1.19] with CRI-O runtime. New features, changes, and known issues that pertain to
+{product-title} {product-version} are included in this topic.
+
+//Red Hat did not publicly release {product-title} 4.6.0 as the GA version and,
+//instead, is releasing {product-title} 4.6.1 as the GA version.
+
+{product-title} {product-version} clusters are available at
+https://cloud.redhat.com/openshift. The {cloud-redhat-com}
+application for {product-title} allows you to deploy OpenShift clusters to
+either on-premise or cloud environments.
+
+{product-title} {product-version} is supported on Red Hat Enterprise Linux 7.7 or
+later, as well as {op-system-first} 4.6.
+
+You must use {op-system} for the control plane, which are also known as master machines, and
```

Comment:
The wording is off, perhaps: "You must use {op-system} _machines_ for the control plane"?

Reply:
also suggest:
s/, and can use either/, and you can use either/